docs/src/background/other_projects.rst

```diff
@@ -0,0 +1,360 @@
+
+Finally, we note that projects are listed in alphabetical order.
+
+*********
+Autocmake
+*********
+
+- Website: `<http://autocmake.org/>`_
+- Source: `<https://github.com/dev-cafe/autocmake>`_.
+- Development status: active.
+- User community: small.
+
+The motivation for Autocmake :cite:`autocmake` was to avoid copy/pasting CMake
+build systems
+across projects and instead generate them from a configuration file. Autocmake
+is written in a mix of Python and CMake. The Python parts focus on the
+generation, whereas the CMake modules largely focus on finding specific
+dependencies (e.g., there are CMake modules for Boost, Python, GoogleTest),
+though there are also some which provide useful features (e.g, colored CMake
+messages, and a safe guard for avoiding in-source builds).
+
+Ultimately, the use of the generator means that projects which use Autocmake
+suffer from the problems described :ref:`here <why_not_a_generator>`. The
+documentation also suggests (see `here <https://tinyurl.com/mr49kffb>`__ for
+example) that parts of the build system are Python-based, and that users can not
+use established CMake workflows. We were not able to readily identify if
+projects which use Autocmake can have dependencies which also use Autocmake,
+though we suspect that recursion IS allowed.
+
+***
+BLT
+***
+
+- Website: `<https://llnl-blt.readthedocs.io/en/develop/>`_
+- Source: `<https://github.com/llnl/blt>`_
+- Development status: active.
+- User community: large.
+
+BLT :cite:`blt` appears to stand for "Build, Link, and Test", though the README
+suggests
+there can be "-ing" suffixes as well. BLT is designed to make it easy to
+declare libraries/executables and link them to dependencies commonly encountered
+in :term:`HPC`. Since it's native CMake, users can further customize their
+build system by writing their own CMake infrastructure.
+
+Ultimately, BLT is probably very useful if you want to build an :term:`HPC`
+application, with minimal dependencies (aside from those found in the standard
+:term:`HPC` toolkit). However, BLT provides very minimal support for finding
+other dependencies, and as far as we can tell, no support for building other
+dependencies. That said, the size of the user community suggests that, despite
+these limitations there is quite a bit of demand for BLT.
+
+*****
+Cinch
+*****
+
+- Website: N/A
+- Source: `<https://github.com/laristra/cinch>`_
+- Development status: inactive.
+- User community: small.
+
+Like other projects on this page, Cinch :cite:`cinch` is designed to cut back
+on the amount of coding needed to write a CMake-based build system. The
+documentation is a bit sparse, but it appears that Cinch provides CMake bindings
+that wrap a Python tool
+`cinch-utils <https://github.com/laristra/cinch-utils>`_. The build system
+developer then writes a CMake-based build system in terms of the CMake bindings.
+
+In terms of features, Cinch seems to be primarily interested in support
```

Comment:
```suggestion
In terms of features, Cinch seems to be primarily interested in
```

This didn't quite make sense reading as "...primarily interested in support facilitating the building...".